### PR TITLE
Dynamo batch get items

### PR DESCRIPTION
**Pull request by nimirium**

### CODE REVIEW

File: sosww/components/dynamo_db.py [hunk 1]
```diff
@@ -316,6 +317,72 @@ def _build_scan_iterator(self, attrs=None, table_name=None, strict=True):
         return response_iterator
 
 
+    def batch_get_items_one_table(self, keys_list, table_name=None, max_retries=0, retry_wait_base_time=0.2):
```
Comment (ngr) [inline sosww/components/dynamo_db.py]: You probably wanted 1 here?
There is some confusion between `tries` and `retries`

File: sosww/components/dynamo_db.py [hunk 2]
```diff
@@ -316,6 +317,72 @@ def _build_scan_iterator(self, attrs=None, table_name=None, strict=True):
...
+        """
+        Gets a batch of items from a single dynamo table.
+        Only accepts keys, can't query by other columns.
+
+        :param list keys_list: A list of the keys of the items we want to get.
+                               Ex: [{'hash_col': '1, 'range_col': 2}, {'hash_col': 3, 'range_col': 4}, ...]
```
Comment (ngr) [inline sosww/components/dynamo_db.py]: Why is this a list?
Please explain the behavior and logic better in documentation (and comments in code).

File: sosww/components/dynamo_db.py [hunk 3]
```diff
@@ -316,6 +317,72 @@ def _build_scan_iterator(self, attrs=None, table_name=None, strict=True):
...
+
+        # Convert given keys to dynamo syntax
+        query_keys = [self.dict_to_dynamo(item) for item in keys_list]
+
+        batch_get_item_query = {
+            'RequestItems': {
+                table_name: {
+                    'Keys': query_keys
+                }
+                # Just so you know - you could add another query for another table here.
```
Comment (ngr) [inline sosww/components/dynamo_db.py]: give commented example of how to do that please.

File: sosww/components/dynamo_db.py (hunk 1 above)
Comment (nimirium) [reply sosww/components/dynamo_db.py]: I try once and I want no more retries

File: sosww/components/dynamo_db.py (hunk 3 above)
Comment (nimirium) [reply sosww/components/dynamo_db.py]: I'll just remove this comment...

File: sosww/components/dynamo_db.py (hunk 2 above)
Comment (nimirium) [reply sosww/components/dynamo_db.py]: What would you use instead of a list?

**Pull request merged**